### ISSUE TITLE
conv1d precision issue.

### ISSUE THREAD

**Issue by sywangyi** [intel/torch-xpu-ops#2391]
### 🐛 Describe the bug

```
import torch
seed = 42
torch.manual_seed(seed)
torch.xpu.manual_seed_all(seed)
in_channel = 128
out_channel = 256
kernel_size = 8
stride = 4
padding = 2

conv = torch.nn.Conv1d(in_channel, out_channel, kernel_size, stride, padding).to("xpu")
hidden_state = torch.randn([1, 128, 46880]).to("xpu")
with torch.no_grad():
    output = conv(hidden_state)
conv_cpu = conv.cpu()
hidden_state_cpu = hidden_state.cpu()
with torch.no_grad():
    output_cpu = conv_cpu(hidden_state_cpu)
torch.testing.assert_close(output_cpu, output.cpu(), rtol=0, atol=0)
```

### Versions

case pytest -rA tests/models/dac/test_modeling_dac.py::DacIntegrationTest::test_integration_0_dac_16khz fail in xpu, because of conv1d precision, may the conv1d to cpu could make the issue pass.
I write a simple test.  and try to make sure if the conv1d precision is expected in xpu.
using the script
Greatest absolute difference: 4.0531158447265625e-06 at index (0, 3, 146)
Greatest relative difference: 28.076923370361328 at index (0, 78, 6176) is this accepted?

**Comment by LuFinch** [comment]
Both CPU and XPU use OneDNN for Conv1D but they have different implementation. 

I refer Pytorch Conv1D [UT](https://github.com/pytorch/pytorch/blob/a5436a5e8e4ee42d1debf52c2786c7ae0043a434/test/nn/test_convolution.py#L2136 )'s atol and rtol and run your case, it could pass.

I think rtol=0 and atol=0 is unreasonable for float point accuracy check. Is rtol=2e-5, atol=2e-5 good for you?
